.github/workflows/mac_build.yml

```
@@ -0,0 +1,39 @@
+on:
+  push:
+    branches:
+      - main
+      - macbuild  # just for testing
+jobs:
+  mac_build:
+    runs-on: macos-11.0
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: "3.9"
```

Using a brew-installed python might fix the build (https://github.com/actions/setup-python/issues/58):

```suggestion
    - run: brew install python@3.9
```